src/generators/legacy-json/utils/buildSection.mjs

```diff
@@ -0,0 +1,543 @@
+    }
+
+    // Add this section to our parent
+    parentSection[sectionTypePlurals[section.type]].push(section);
+
+    return section;
+  };
+
+  /**
+   * Grabs stability number & text and adds it to the section
+   * @param {import('../types.d.ts').Section} section
+   */
+  const parseStability = section => {
+    let needsCompressing = false;
+
+    // Remove metadata not directly the inferable from the markdown
+    nodes.forEach((node, i) => {
+      if (
+        node.type === 'blockquote' &&
+        node.children.length === 1 &&
+        node.children[0].type === 'paragraph' &&
+        nodes.slice(0, i).every(node => node.type === 'list')
+      ) {
+        const text = textJoin(node.children[0].children);
+        const stability = /^Stability: ([0-5])(?:\s*-\s*)?(.*)$/s.exec(text);
+        if (stability) {
+          section.stability = parseInt(stability[1], 10);
+          section.stabilityText = stability[2].replaceAll('\n', ' ').trim();
+
+          delete nodes[i];
+          needsCompressing = true;
+        }
+      }
+    });
+
+    if (needsCompressing) {
+      // Compress to remove the holes left by deletions
+      nodes = nodes.filter(() => true);
+    }
+  };
+
+  /**
+   *
+   * @param {import('../types.d.ts').Section} section
+   */
+  const parseListIfThereIsOne = section => {
+    const list =
+      nodes.length && nodes[0].type === 'list' ? nodes.shift() : null;
+    if (!list) {
+      return;
+    }
+
+    /**
+     * @type {Array<import('../types.d.ts').List>}
+     */
+    const values = list ? list.children.map(child => parseListItem(child)) : [];
+    switch (section.type) {
+      case 'ctor':
+      case 'classMethod':
+      case 'method': {
+        section.signatures = [parseSignature(section.textRaw, values)];
+
+        break;
+      }
+
+      case 'property': {
+        if (!values.length) {
+          break;
+        }
+
+        const signature = values[0];
+        signature.textRaw = `\`${section.name}\` ${signature.textRaw}`;
+
+        for (const key in signature) {
+          if (!signature[key]) {
+            continue;
+          }
+
+          if (key === 'type') {
+            // We'll set propertySigType to type at the end since we still need the
+            //  original type for a few more checks
+            section.propertySigType = signature.type;
+          } else {
+            section[key] = signature[key];
+          }
+        }
+
+        break;
+      }
+
+      case 'event':
+        section.params = values;
+        break;
+
+      default:
+        // List wasn't consumed, add it back
+        nodes.unshift(list);
+    }
+  };
+
+  /**
+   * @param {import('../types.d.ts').Section} section
+   */
+  const addDescription = section => {
+    if (nodes.length === 0) {
+      return;
+    }
+
+    if (section.desc) {
+      section.shortDesc = section.desc;
+    }
+
+    // Render the description as if it was html
+    section.desc = unified()
```

Comment:
No need to do this, can you simply call this? https://github.com/nodejs/api-docs-tooling/blob/main/src/utils/remark.mjs#L20

We don't need remark-html, remark-rehype is good enough and does the same :)

(remark-rehype then rehype-stringify)

Reply:
You can also remove the depenendecy of remark-html